plugins/inputs/statsd/statsd.go

```diff
@@ -348,7 +348,7 @@ func (s *Statsd) parseStatsdLine(line string) error {
 				return errors.New("Error Parsing statsd line")
 			}
 			m.floatvalue = v
-		case "c", "s":
+		case "c", "s", "m", "mr":
```

Comment:
@sparrc waiting for more information about those - https://discourse.chef.io/t/super-hero-protocol-estatsd-format/7758